components/photoview/comments.js

```diff
@@ -14,7 +14,7 @@ export default class CommentView extends React.Component {
 
 	this.state = {
 		comments: this.ds,
-		commentText: 'Make a comment',
+		commentText: 'Write a comment...',
```

Comment:
We should talk about this change but if we're going to change it, there are more places in this file we need to change it as well.

Reply:
Reverted